pyproject.toml

```diff
@@ -9,7 +9,7 @@ style = "semver"
 pattern = "default-unprefixed"
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.7,<=3.12"
```

Comment:
Does it mean that we can now develop in python 3.10?? 🤩

Reply:
As long as the code itself is still compatible with Python 3.7 -> Sure 😈